docs/installation-on-k3d.md

```diff
@@ -0,0 +1,131 @@
+## Local k3d setup for local lifecycle manager & keda-manager
+
+1. Clone the project.
+
+   ```bash
+   git clone https://github.com/kyma-project/keda-manager.git && cd keda-manager/
+   ```
+
+2. Provide the k3d cluster.
+
+   ```bash
+   kyma provision k3d
+   ```
+
+3. Build and push the Keda Manager image.
+
+   ```bash
+   make module-image IMG_REGISTRY=localhost:5001/unsigned/operator-images IMG=localhost:5001/keda-manager-dev-local:0.0.1
+   ```
+
+4. Build and push the Keda module.
+
+   ```bash
+   make module-build IMG=k3d-kyma-registry:5001/keda-manager-dev-local:0.0.1 MODULE_REGISTRY=localhost:5001/unsigned
+   ```
+
+5. Verify if the module and the manager's image are pushed to the local registry.
+
+   ```bash
+   curl localhost:5001/v2/_catalog
+   ```
+You should get a result similar to this example:
+
+   ```json
+   {"repositories":["keda-manager-dev-local","unsigned/component-descriptors/kyma-project.io/module/keda"]}
+   ```
+6. Inspect the generated module template.
+
+> **NOTE:** The following sub-steps are temporary workarounds.
+
+Edit `template.yaml` and:
+- change `target` to `control-plane`
+
+   ```yaml
+   spec:
+    target: control-plane
+    ```
+> **NOTE:** This is required in the single-cluster mode only.
+
+- change the existing repository context in `spec.descriptor.component`:
+
+   ```yaml
+   repositoryContexts:      
+     - baseUrl: k3d-kyma-registry.localhost:5000/unsigned
+       componentNameMapping: urlPath
+       type: ociRegistry
+   ```
+
+> **NOTE:** Because Pods inside the k3d cluster use the docker-internal port of the registry, it tries to resolve the registry against port 5000 instead of 5001. K3d has registry aliases, but `module-manager` is not part of k3d and does not know how to properly alias k3d-kyma-registry.localhost:5001.
```

Comment:
module-manager is going to be merged with lifecycle-manager, so maybe we need to change/delete this note.

Reply:
```suggestion

```